### PR TITLE
Limitations

### PR DESCRIPTION
**Pull request by Arjun-thakor-005**
Nice .

### CODE REVIEW

**Review by Adarsh8763 — COMMENTED** [review]
Hey, I am doing this because I am an animal lover.

**Comment by Arjun-thakor-005** [comment]
Ok that's nice . Pls accept my pull request

**Comment by Adarsh8763** [comment]
Hey , I am new in this and i don't no how to accept someone's pull request . So, can u pls suggest me how to do that

**Comment by Arjun-thakor-005** [comment]
Ok wait

**Comment by Arjun-thakor-005** [comment]
To *accept* the *pull request*, click the *Pull Requests* tab to see a
summary of pending *pull requests*. If you are happy with the changes,
click *Merge Pull request* to *accept* the *pull request* and perform the
*merge*. You can add in a comment if you want

**Comment by Arjun-thakor-005** [comment]
Ok accept my pull request

**Review by Adarsh8763 — COMMENTED** [review]
It's ok

**Comment by Adarsh8763** [comment]
It is not showing any pending pull request.

**Comment by Arjun-thakor-005** [comment]
Wait

**Comment by Arjun-thakor-005** [comment]
Ok

**Comment by Arjun-thakor-005** [comment]
I have done another request ok
 If also not happen then next time see u ..👍

**Comment by Adarsh8763** [comment]
Is writing 'review' is necessary
Now also i am not able to see any pending pull request

**Comment by Arjun-thakor-005** [comment]
Ok
Next time

File: Limitations. [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Limitations.py
+U should write u code limitations
```
Comment (Adarsh8763) [inline Limitations.:2]: U should write your code limit.

**Review by Adarsh8763 — APPROVED** [review]

**Pull request merged**